### PR TITLE
preprocess.py

### PR DESCRIPTION
**Pull request by rbechto2**
Add preprocess.py that combines all the preprocessing subfunctions

### CODE REVIEW

**Comment by rbechto2** [comment]
Attached to issue: Implementation for pyAutomagic preprocessing #18
Also, updated the preprocess function, added tests, and added a demo of the function's use.

File: pyautomagic/preprocessing/rpca.py [hunk 1]
```diff
@@ -1,103 +1,100 @@
 import numpy as np
 from sklearn.decomposition import TruncatedSVD
```
Comment (adam2392) [inline pyautomagic/preprocessing/rpca.py]: Is this being used?

Comment (adam2392) [reply pyautomagic/preprocessing/rpca.py]: If not, should probably take it out.

**Comment by saulmezac** [comment]
I approved the changes, but before merging the PR: delete your dummy Project, dummy Subject, dummy rateQuality and just add what you need to test_block

**Pull request merged**